modules/user_accounts/php/edit_user.class.inc

```diff
@@ -871,12 +871,13 @@ class Edit_User extends \NDB_Form
                 )
             );
 
-            $dateOptions = array(
+            // active time windows
+            $dateFormat     = $config->getSetting('dateDisplayFormat');
+            $dateOptions    = array(
                 'language'       => 'en',
-                'format'         => 'YMd',
+                'format'         => $dateFormat,
```

Comment:
i believe this date option is for a date input element. this PR wants to format date displays i.e. date shown as string